README.adoc

```diff
@@ -86,7 +87,7 @@ include::complete/src/main/java/hello/Application.java[]
 
 include::https://raw.githubusercontent.com/spring-guides/getting-started-macros/master/spring-boot-application.adoc[]
 
-The {EnableAsync}[`@EnableAsync`] annotation switches on Spring's ability to run `@Async` methods in a background thread pool. This class also customizes the used `Executor`. In our case, we want to limit the number of concurrent threads to 2 and limit the size of the queue to 500. There are {Executor}[many more things you can tune]. By default, a `SimpleAsyncTaskExecutor` is used.
+The {EnableAsync}[`@EnableAsync`] annotation switches on Spring's ability to run `@Async` methods in a background thread pool. This class also customizes the used `Executor` by defining a new bean. Here the method is named `taskExecutor` since this is the {EnableAsyncJavadoc}[specific method name Spring will search for]. In our case, we want to limit the number of concurrent threads to 2 and limit the size of the queue to 500. There are {Executor}[many more things you can tune]. By default, a `SimpleAsyncTaskExecutor` is used.
```

Comment:
> By default, a `SimpleAsyncTaskExecutor` is used.

This seems important, but it's kind of just stuck on the end of the paragraph with no context. It would be better to explain that "by default" means "if we hadn't defined an `Executor` bean", maybe? In any case you can leave that to me when I merge if you agree, or else update the patch. Thanks.